packages/venia-concept/src/components/Checkout/flow.css

```diff
@@ -10,6 +10,7 @@
     bottom: 5.5rem;
     box-shadow: 0 -1px rgb(var(--venia-border));
     display: grid;
+    grid-template-columns: 50% 50%;
```

Comment:
Don't think we want the checkout options to look like this:
![image](https://user-images.githubusercontent.com/6378569/49319729-21e85e00-f4b3-11e8-9d2c-cec14f0d346b.png)